### ISSUE TITLE
The split of the Stanford Cars dataset

### ISSUE THREAD

**Issue by ppanzx** [htdt/hyp_metric#9]
Thank you for your commendable efforts in your work. I have a question regarding the split of the Stanford Cars dataset, which comprises 16,185 images representing 196 car models, in comparison to other tasks. 

In most metric-learning literature, the dataset split is described as follows: "The first 98 classes (8,054 images) are used for training, and the remaining 98 classes (8,131 images) are held out for testing."

However, the split mentioned in the [Torchvision](https://pytorch.org/vision/stable/_modules/torchvision/datasets/stanford_cars.html#StanfordCars) documentation states that "The data is split into 8,144 training images and 8,041 testing images, with an approximately 50-50 split for each class.", the training and testing split of which is different from your work. 

Unfortunately, the [official website](https://ai.stanford.edu/~jkrause/cars/car_dataset.html) is currently inaccessible, leaving me uncertain about the specific split used in this implementation.

Could you kindly provide me with a detailed split list (rather than the raw images) used in your implementation of the Stanford Cars dataset?

Thank you for your attention to this matter.
